### PR TITLE
[IMP] tutorials: add example code for LazyComponent

### PR DESCRIPTION
**Pull request by agau-odoo**
js framework/Build a Dashboard/part 8 (LazyComponent)

There is no explicit instructions on how to use `LazyComponent`.

- add an example code snippet for minimal use of `LazyComponent`
- add the `.js` extension to the file you're supposed to create to make it a bit more intuitive
- fix `actions` registry name (missing an s)
- reword instructions in a more straightforward way

### CODE REVIEW

**Review by fdardenne — COMMENTED** [review]
Hello 👋 
As I said in the channel, it's an enterprise link that is not accessible by everyone. It'll maybe create frustration for those without access.

However it's still an improvement over the old one, especially for partners and Odoo newbies. Maybe indicate that it's a link to an enterprise repo?

In the future, we should surely include a snippet of how LazyComponent is used, or make a documentation on this component.

**Comment by agau-odoo** [comment]
Let's make the future now! ✨
I've updated my PR to include a code snippet, fix a few quirks and improve the wording.

Please let me know what you think

**Review by fdardenne — APPROVED** [review]
Looks good to me, thank you!

File: content/developer/tutorials/discover_js_framework/02_build_a_dashboard.rst [hunk 1]
```diff
@@ -322,17 +322,30 @@ of our users. In that case, it could make sense to lazy load our dashboard, and
...
+
+   // example_action.js
+   export class ExampleComponentLoader extends Component {
+       static components = {LazyComponent};
+       static template = xml`
+           <LazyComponent bundle="'example_module.example_assets'" Component="'ExampleComponent'" />
+       `;
+   }
+
+   registry.category("actions").add("example_module.example_action", ExampleComponentLoader);
```
Comment (AntoineVDV) [inline content/developer/tutorials/discover_js_framework/02_build_a_dashboard.rst:340]: When possible, use the `example` admonition for examples so you can tell them apart from the rest of the content.
```suggestion
.. example::

   .. code-block:: javascript

      export class ExampleComponentLoader extends Component {
          static components = {LazyComponent};
          static template = xml`
              <LazyComponent bundle="'example_module.example_assets'" Component="'ExampleComponent'" />
          `;
      }

      registry.category("actions").add("example_module.example_action", ExampleComponentLoader);
```

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+ delegate=fdardenne

Reply with `@robodoo r\+` (remove the `\` from the command to make it work) when you're ready to merge :)

**Comment by fdardenne** [comment]
@robodoo r+

**Pull request closed without merging**